include/up-cpp/communication/RpcServer.h

```diff
@@ -114,6 +116,9 @@ struct RpcServer {
 
 	/// @brief Handle to the connected callback for the RPC method wrapper
 	transport::UTransport::ListenHandle callback_handle_;
+
+	/// @brief Format of the payload that will be expected in responses
+	std::optional<v1::UPayloadFormat> expected_payload_format_;
```

Comment:
Minor: move this before the `callback_handle_` member. This ensures the handle is destroyed before any other members are deallocated.

Reply:
addressed